exonum/Cargo.toml

```diff
@@ -58,6 +58,7 @@ exonum_flamer = { path = "../3rdparty/flamer", version = "0.1.6", optional = tru
 pretty_assertions = "0.5.0"
 tempdir = "0.3.5"
 lazy_static = "1.0.0"
+compiletest_rs = "*"
```

Comment:
Actually I guess it's a bad practice to use `*` instead numeric version

Reply:
Actually I'm not sure about additional dependency. Do we really need it? Let's discuss it privately with core team.